Microsoft.Toolkit.Uwp.UI.Controls.Input/RadialGauge/RadialGauge.cs

```diff
@@ -193,34 +193,10 @@ private void ThemeListener_ThemeChanged(ThemeListener sender)
             OnColorsChanged();
         }
 
-        private void RadialGauge_KeyDown(object sender, KeyRoutedEventArgs e)
-        {
-            double step = SmallChange;
-            var ctrl = Window.Current.CoreWindow.GetKeyState(VirtualKey.Control);
-            if (ctrl.HasFlag(CoreVirtualKeyStates.Down))
-            {
-                step = LargeChange;
-            }
-
-            step = Math.Max(StepSize, step);
-            if ((e.Key == VirtualKey.Left) || (e.Key == VirtualKey.Down))
-            {
-                Value = Math.Max(Minimum, Value - step);
-                e.Handled = true;
-                return;
-            }
-
-            if ((e.Key == VirtualKey.Right) || (e.Key == VirtualKey.Up))
-            {
-                Value = Math.Min(Maximum, Value + step);
-                e.Handled = true;
-            }
-        }
-
         private void RadialGauge_Unloaded(object sender, RoutedEventArgs e)
         {
             // Unregister event handlers.
-            KeyDown -= RadialGauge_KeyDown;
+            KeyboardAccelerators.Clear();
```

Comment:
Pretty sure we need to do this in `OnApplyTemplate` as it can be called multiple times without the Unloaded event being called. (Actually the same for all these events.) Otherwise, we're going to have duplicate event registrations/firings or not free up memory.

@Sergio0694 just wanted to check, since the lambda event hook is encapsulated in the `KeyboardAccelerator` object, and we clear that collection with this call, then there won't be any references, so we'll be in the clear, right? (i.e. it's a simpler pattern than what we normally have to do.)